### DISCUSSION TITLE
PhotoPrism goes "Offline" when converting CR2 files

### DISCUSSION THREAD

**Discussion by DylanTusler** [photoprism/photoprism#4572] [Q&A]
New install, mostly everything is working great. I have a library of about 40k images, with a small number of .CR2 files from a Canon DSLR camera scattered amongst them.

Everything is being done on a single Windows machine running PhotoPrism under Docker with the suggested minimum swap file and ram sizes.

Running indexing goes fine until it hits a folder with CR2 images in it. Then it converts a few, then stalls.

2024-10-12 12:09:18 INFO convert: converting IMG_7688.CR2 to IMG_7688.CR2.jpg (darktable-cli)

While converting the last image before stalling, it pops up an Offline alert (sometimes a toast message saying "Request failed - are you offline?" or the Offline indicator in bottom left corner of the UI.) Nothing is, as far as I can tell, offline. If I restart indexing, it will get past the file it stalled on last time and go a little further. I get about 6 or 7 conversions in before it stalls again.

Any thoughts on what may be wrong?

![image](https://github.com/user-attachments/assets/0cb802de-3396-4030-823c-7c888134141f)

**Comment by graciousgrey** [comment]
You can find our full troubleshooting checklist for this here:

- https://docs.photoprism.app/getting-started/troubleshooting/#fatal-server-errors

As this happens when converting RAW files I would recommend checking this first:

- https://docs.photoprism.app/getting-started/troubleshooting/docker/#adding-swap

**Comment by tilusnet** [comment]
This post lead me to solve a related issue I was having. 

Lengthy Photoprism API requests - longer than 60 seconds - are likely to get "cut off" by reverse proxies like Traefik. 
My use case was related to uploading video files - few hundred megs took longer than 1 minute, and these always failed:

```
time="2025-12-16T22:30:55Z" level=debug msg="api-v1: abort /hunhun/api/v1/users/:uid/upload/:token with code 400 (upload failed)"
time="2025-12-16T22:30:55Z" level=debug msg="server: POST /hunhun/api/v1/users/uswvkc5ofkz1nup9/upload/ke9ckk (400) [1m0.003728445s]"
time="2025-12-16T22:31:57Z" level=error msg="upload: unexpected EOF"
```
(notice the exact cut off after 1 minute)
